src/config.ts

```diff
@@ -160,7 +160,7 @@ export function getKamelRuntimeVersionConfig(): string | undefined {
 
 export async function setKamelRuntimeVersionConfig(value : string) : Promise<void> {
 	const configuration = vscode.workspace.getConfiguration();
-	return await configuration.update(RUNTIME_VERSION_KEY, stripLeadingV(value), vscode.ConfigurationTarget.Global);
+	return configuration.update(RUNTIME_VERSION_KEY, stripLeadingV(value), vscode.ConfigurationTarget.Global);
```

Comment:
If await is no longer needed, do we need to return a Promise?

Reply:
the `await` is not needed for returned values inside the `async` method which is returning a `Promise`.

more information in the description of the issue on Sonar: https://sonarcloud.io/organizations/camel-tooling/rules?open=typescript%3AS4326&rule_key=typescript%3AS4326